packages/host/src/components/Initialise/Initialise.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { useEffect } from 'react';
+import {
+  useTranslation,
+  LoadingButton,
+  Box,
+  Typography,
+  AlertIcon,
+  useHostContext,
+  SaveIcon,
+} from '@openmsupply-client/common';
+import { LoginTextInput } from '../Login/LoginTextInput';
+import { InitialiseLayout } from './InitialiseLayout';
+import { useInitialiseForm } from './hooks';
+
+export const Initialise = () => {
+  const t = useTranslation('app');
+  const { setPageTitle } = useHostContext();
+
+  const {
+    isValid,
+    isLoading,
+    password,
+    siteId,
+    url,
+    username,
+    onSave,
+    setPassword,
+    setUsername,
+    setSiteId,
+    setUrl,
+    error,
+  } = useInitialiseForm();
+
+  useEffect(() => {
+    setPageTitle(`${t('app.initialise')} | ${t('app')} `);
+  }, []);
+
+  return (
+    <InitialiseLayout
+      UsernameInput={
+        <LoginTextInput
+          fullWidth
+          label={t('label.settings-username')}
+          value={username}
+          disabled={isLoading}
+          onChange={e => setUsername(e.target.value)}
+          inputProps={{
+            autoComplete: 'username',
+          }}
+          autoFocus
+        />
+      }
+      PasswordInput={
+        <LoginTextInput
+          fullWidth
+          label={t('label.settings-password')}
```

Comment:
Got confused here and entered the user credentials and not the central server account. Maybe better to include this into the label (maybe also helpful for the URL).

Maybe just me, but I also ignored the left part of the screen with the description of the init page, probably assuming that it doesn't contain information about the screen but just general mSupply info. Maybe move the description into the form to make it clearer that this is the initial setup screen? Maybe also a big header like "Initial server setup" and/or different colour for the init screen....